### DISCUSSION TITLE
Test runs with coupled configurations on HAFS supported platforms

### DISCUSSION THREAD

**Discussion by BijuThomas-NOAA** [hafs-community/HAFS#93] [General]
@BinLiu-NOAA and @evankalina 

Here is the status of the test runs with 15 coupling configurations:

**Hera**: Succeded

**Wcoss-Cray**: Succeded with the following changes:

- Added the entities FORECAST_RESOURCES_regional_40x30io1x96_ocn60_wav60_omp2 and FORECAST_RESOURCES_regional_40x30io1x96_med60_wav60_omp2 in wcoss_cray.ent
- The environment variable G2_LIB4 pointing to a non-existing path. Added correct path for G2_LIB4 in modulefiles/modulefile.hafs.wcoss_cray. Opened an issue [**310**](https://github.com/NOAA-EMC/hpc-stack/issues/310)  at EMC/hpc-stack.

**Wcoss-Dell**: Succeded with the following changes:

- Added the entities FORECAST_RESOURCES_regional_40x30io1x96_ocn60_wav60_omp2 and FORECAST_RESOURCES_regional_40x30io1x96_med60_wav60_omp2 in wcoss_dell.ent

Do you want me to push these changes or open a PR?

**Comment by BinLiu-NOAA** [comment]
@BijuThomas-NOAA, Thanks for these modifications and updates! Please commit these changes directly into the feature/hafs_coupleww3 branch. Thanks!
